### DISCUSSION TITLE
GitHub Projects - Recommended Best Practices / Showcases

### DISCUSSION THREAD

**Discussion by tsteckenborn** [community/community#38066] [Projects and Issues]
Hello everyone,

Do you know best practices (besides the [documentation](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/best-practices-for-projects)) or successful examples of using GitHub Projects as a small team? Have you seen any recommendable videos or showcases on this topic?

Are there any approaches you recommend?
Is there anything that didn't work out as planned?

We would greatly appreciate any lessons learned and recommendations for use.

**Comment by david-engelmann** [comment]
[This guide is more complete](https://docs.github.com/en/issues/planning-and-tracking-with-projects). It's a fairly new feature, but very exciting.
